### PR TITLE
controlsState: enter pre-enable when engaging at a standstill with brake

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/interfaces.py [hunk 1]
```diff
@@ -124,8 +124,8 @@ def create_common_events(self, cs_out, extra_gears=None, pcm_enable=True):
       events.add(EventName.wrongCarMode)
     if cs_out.espDisabled:
       events.add(EventName.espDisabled)
-    if cs_out.gasPressed:
-      events.add(EventName.gasPressed)
+    if cs_out.gasPressed or cs_out.brakePressed:
```
Comment (adeebshihadeh) [inline selfdrive/car/interfaces.py:127]: add the standstill check here

Comment (sshane) [reply selfdrive/car/interfaces.py:127]: We only enter pre-enable state if there's no NO_ENTRY events and an ENABLE event, so `pedalPressedPreEnable` only has any affect when `pedalPressed` isn't sent

Comment (adeebshihadeh) [reply selfdrive/car/interfaces.py:127]: yeah, the state machine will handle it, but better to be explicit

**Comment by sshane** [comment]
Vehicle-specific issues make this difficult to do (Honda operates on brake pressure, so when you release brake no matter how fast, you move forward)

**Pull request closed without merging**